packages/components/src/components/toast/toast-item/toast-item.tsx

```diff
@@ -45,6 +46,12 @@ export class ToastItem {
     throwIfRootNodeIsNotOneOfKind(this.host, ['p-toast']);
   }
 
+  public componentDidRender(): void {
+    if (getHasNativePopoverSupport()) {
+      this.host.showPopover();
```

Comment:
How does the focus behave in case a flyout with `<dialog/>` is opened which then triggers a toast?

Reply:
As discussed: if the Toast is initialized inside of the `dialog` it is interactable. If it is initialized outside of `dialog` (e.g. end of `body`) then no interaction can be done. See also conversation on https://github.com/whatwg/html/issues/9936